src/bioregistry/align/ncbi.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+
+"""Align NCBI with the Bioregistry."""
+
+from typing import Any, Dict, List
+
+from bioregistry.align.utils import Aligner
+from bioregistry.external.ncbi import get_ncbi
+
+__all__ = ['NcbiAligner']
+
+
+class NcbiAligner(Aligner):
+    """Aligner for NCBI xref registry."""
+
+    key = 'ncbi'
+    getter = get_ncbi
+    curation_header = ('name', 'generic_urls', 'example')
+
+    def prepare_external(
+        self,
+        external_id: str,
+        external_entry: Dict[str, str],
+    ) -> Dict[str, Any]:
+        """Stub for preparing NCBI xref data to be added to the bioregistry.
+
+        The NCBI getter does all the necessary processing, so this method is effectively an identity.
+
+        :param external_id: the key of the entry in the incoming data
+        :param external_entry: the actual entry data from the incoming data
+
+        :returns: the processed entry data (in this case, no actual processing is done)
+        """
+        return external_entry
```

Comment:
maybe you should just change this in the base class to be the default implementation